app/services/katello/pulp/pulp_content_unit.rb

```diff
@@ -46,7 +46,7 @@ def self.unit_handler
         Katello.pulp_server.extensions.send(self.name.demodulize.underscore)
       end
 
-      def self.pulp_units_batch_all(unit_ids = nil, page_size = SETTINGS[:katello][:pulp][:bulk_load_size])
+      def self.pulp_units_batch_all(unit_ids = nil, page_size = ::Setting::Content.find_by(name: "bulk_load_size").value)
```

Comment:
Can't you just do  Setting[:bulk_load_size] ?

Reply:
Ah yeah, that should work.